Makefile

```diff
@@ -39,18 +41,7 @@ before-build:
 
 # Collects all dependencies and then calls update
 .PHONY: collect
-collect: imports fields go-generate create-docs notice
-
-# Generates imports for all modules and metricsets
-.PHONY: imports
-imports:
-	@mkdir -p include
-	@mkdir -p processor
-
-.PHONY: fields
-fields:
-	@cat _meta/fields.common.yml > _meta/fields.generated.yml
-	@cat processor/*/_meta/fields.yml >> _meta/fields.generated.yml
```

Comment:
Changes are aligned with changes made when updated to beats framework in master, see https://github.com/elastic/apm-server/pull/987/files#diff-b67911656ef5d18c4ae36cb6741b7965.